### ISSUE TITLE
Projectile is spitting out erros

### ISSUE THREAD

**Issue by sunnyhasija** [hlissner/doom-emacs#3677]
**What did you expect to happen?**
...
I expected not to get projectile errors

**What actually happened?**
...
The following error is spit out every time I try to do something in doom .
```projectile-merge-known-projects: Wrong type argument: sequencep, nil/\.emacs\.d/```

**Additional details:**
- Include a link to your private config

private config is located at https://github.com/sunnyhasija/DOOMEmacs

- Include any warnings or errors logged to \*Messages\* (use `M-x
  view-echo-area-messages` to see it).

<details><pre>
If an error message is involved include a backtrace of it.
``` projectile-merge-known-projects: Wrong type argument: sequencep, nil/\.emacs\.d/
mwheel-scroll: Beginning of buffer
Mark set
projectile-merge-known-projects: Wrong type argument: sequencep, nil/\.emacs\.d/```


</pre></details>

**System information:**
<details><pre>
```
emacs   version    26.3
        features   XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND DBUS GSETTINGS GLIB NOTIFY ACL LIBSELINUX GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS XWIDGETS LCMS2
        build      Jan 28, 2020
        buildopts  (--build=x86_64-redhat-linux-gnu --host=x86_64-redhat-linux-gnu --program-prefix= --disable-dependency-tracking --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --datadir=/usr/share --includedir=/usr/include --libdir=/usr/lib64 --libexecdir=/usr/libexec --localstatedir=/var --sharedstatedir=/var/lib --mandir=/usr/share/man --infodir=/usr/share/info --with-dbus --with-gif --with-jpeg --with-png --with-rsvg --with-tiff --with-xft --with-xpm --with-x-toolkit=gtk3 --with-gpm=no --with-xwidgets --with-modules build_alias=x86_64-redhat-linux-gnu host_alias=x86_64-redhat-linux-gnu 'CFLAGS=-DMAIL_USE_LOCKF -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection' LDFLAGS=-Wl,-z,relro PKG_CONFIG_PATH=:/usr/lib64/pkgconfig:/usr/share/pkgconfig)
        windowsys  x
        daemonp    server-running
doom    version    2.0.9
        build      HEAD -> develop 15e156294 2020-08-03 02:52:55 -0400
        dir        ~/.doom.d/
system  type       gnu/linux
        config     x86_64-redhat-linux-gnu
        shell      /usr/bin/fish
        uname      Linux 5.7.11-200.fc32.x86_64 #1 SMP Wed Jul 29 17:15:52 UTC 2020 x86_64
        path       (~/.emacs.d/bin/../bin ~/.local/bin ~/bin /sbin /var/lib/flatpak/exports/bin ~/.emacs.d/bin /usr/local/bin ~/.local/bin ~/bin /sbin /var/lib/flatpak/exports/bin ~/.emacs.d/bin /usr/bin /bin /usr/libexec/emacs/26.3/x86_64-redhat-linux-gnu/)
config  envfile    envvar-file
        elc-files  0
        modules    (:completion company ivy :ui deft doom doom-dashboard doom-quit hl-todo modeline ophints (popup +defaults) pretty-code tabs treemacs vc-gutter vi-tilde-fringe window-select workspaces zen :editor (evil +everywhere) file-templates fold snippets word-wrap :emacs dired electric undo vc :checkers syntax spell grammar :tools (eval +overlay) lookup magit pdf :lang emacs-lisp latex markdown (org +roam) python sh :config (default +bindings +smartparens))
        packages   ((org-ref) (interleave) (helm-bibtex) (zotxt) (ox-pandoc) (org-roam-bibtex) (org-noter) (org-roam-server :recipe (:host github :repo org-roam/org-roam-server :files (*))) (org-download) (nov) (rotate) (xkcd) (keycast) (org-super-agenda) (doct :recipe (:host github :repo progfolio/doct)) (org-pretty-table-mode :recipe (:host github :repo Fuco1/org-pretty-table)) (org-pretty-tags) (centaur-tabs) (company-org-roam :recipe (:host github :repo org-roam/company-org-roam)) (org-fancy-priorities) (origami))
        unpin      (org-roam)
        elpa       (org-fancy-priorities)
```
</pre></details>

**Comment by ymarco** [comment]
I had something similar yesterday, turned out my `projectile-known-projects` got somehow assigned to a string (instead of a list of path strings).
I loaded up a backup of my `.emacs.d/.local/cache/projectile.*` files and it was fixed.
So you might want to check if some variable of projectile got assigned a string or something, and roll back the cache if you've got a backup (or delete it if you don't have a backup and you don't mind losing the known project list).
